tools/blocksconvert/scanner/scanner.go

```diff
@@ -213,6 +214,19 @@ func (s *Scanner) running(ctx context.Context) error {
 			}
 
 			reader = newBigtableIndexReader(bigTable.Project, bigTable.Instance, s.logger, s.indexReaderRowsRead, s.indexReaderParsedIndexEntries, s.currentTableRanges, s.currentTableScannedRanges)
+		case "aws-dynamo":
+			cfg := s.storageCfg.AWSStorageConfig
+
+			if cfg.DynamoDB.URL == nil { // TODO: what else do we need?
```

Comment:
Can we remove/solve this TODO?

Reply:
Yeah I think that is the only mandatory setting.